src/software/ai/navigator/obstacle/obstacle_factory.cpp

```diff
@@ -142,31 +141,55 @@ std::vector<ObstaclePtr> ObstacleFactory::createVelocityObstaclesFromTeam(
     return obstacles;
 }
 
-ObstaclePtr ObstacleFactory::createBallObstacle(const Point &ball_position)
+ObstaclePtr ObstacleFactory::createBallObstacle(const Point &ball_position) const
+{
+    return expandForRobotSize(Circle(ball_position, BALL_MAX_RADIUS_METERS + 0.06));
```

Comment:
what's `0.06`?

Reply:
Some extra fudge value that didn't seem worth making a dynamic param. It could probably be removed without consequence

Reply:
If it can be removed I'd prefer that.

Reply:
removed